README.md

```diff
@@ -1,10 +1,60 @@
-# "Finance and support for your business" application
+# Business Support Finder
 
-Frontend faceted search application for finding finance and support options.
+Faceted search application for finding business support and finance options.
 
-Built as a Rails frontend client to a separate API.
+## Screenshots
 
-## Requirements
+![Start Page](https://raw.githubusercontent.com/alphagov/business-support-finder/master/docs/assets/bsf-start.png)
 
-* [Business Support API](https://github.com/alphagov/business-support-api)
-* Ruby 2.1.5
+![Finder Results](https://raw.githubusercontent.com/alphagov/business-support-finder/master/docs/assets/bsf-finder.png)
+
+
+
+## Live examples
+
+- [gov.uk/business-finance-support-finder](https://www.gov.uk/business-finance-support-finder)
+- [gov.uk/business-finance-support-finder/search](https://www.gov.uk/business-finance-support-finder/search)
+
+
+## Technical documentation
+
+This is a Ruby on Rails application that is a frontend to [business-support-api-](https://github.com/alphagov/business-support-api)
+
+It's a precursor to [finder-frontend](https://github.com/alphagov/finder-frontend) application, and should be merged in to it at some point. Currently it uses some filtering behaviour that `finder-frontend` does not support;
+
+- location/postcode filtering
+- a mix of `AND` / `OR` facet filters
+
+The schemes themselves (eg [gov.uk/bridge-to-employment-northern-ireland](https://www.gov.uk/bridge-to-employment-northern-ireland)) are served by [frontend](https://github.com/alphagov/frontend) and managed in [publisher](https://github.com/alphagov/publisher).
+
+### Dependencies
+
```

Comment:
`publishing-api` is now also a dependency (since last week or so), see https://github.com/alphagov/business-support-finder/pull/101.